tutorial/adding-progressive-image-loading.md

```diff
@@ -37,98 +37,98 @@ Go to the file which uses `ResponsiveImage` imported from `@quintype/components`
 1. Create a state, which contains a initial size and blur.
    Ex:
 
-```javascript
-const [perfObj, setPerfObj] = useState({ size: "5vw", blur: 10 });
-```
+   ```javascript
+   const [perfObj, setPerfObj] = useState({size: '5vw', blur: 10});
+   ```
 
-**Note**: Keep the size as minimal as possible. Lower the size, lower is the LCP. Blur can be added or removed based on publisher requirements.
+   **Note**: Keep the size as minimal as possible. Lower the size, lower is the LCP. Blur can be added or removed based on publisher requirements.
 
 2. Update the value of size and blur in the state according to the requirements after an interval, to get a higher quality image.
    Ex:
 
-```javascript
-const [perfObj, setPerfObj] = useState({ size: "5vw", blur: 10 });
-
-useEffect(() => {
-  setTimeout(() => {
-    setPerfObj({ size: "25vw", blur: 0 });
-  }, 2500);
-}, []);
-```
-
-3. Pass the size and blur to the `ResponsiveImage` component. Your final code should look something like this.
-
-```javascript
-...
-import React, { useState, useEffect } from "react";
-import { Link, ResponsiveImage } from "@quintype/components";
-...
-...
-
-function StoryGridStoryItem(props) {
-  const [perfObj, setPerfObj] = useState({ size: "5vw", blur: 10 });
-
-  useEffect(() => {
-    setTimeout(() => {
-      setPerfObj({ size: "25vw", blur: 0 });
-    }, 2500);
-  }, []);
-
-  return (
-    <Link href={`/${props.story.slug}`} className="story-grid-item">
-      <figure className="qt-image-16x9" styleName="story-grid-item-image">
-        <ResponsiveImage
-          slug={props.story["hero-image-s3-key"]}
-          metadata={props.story["hero-image-metadata"]}
-          aspectRatio={[16, 9]}
-          defaultWidth={480}
-          widths={[250, 480, 640]}
-          sizes={perfObj.size}
-          imgParams={{ auto: ["format", "compress"], blur:perfObj.blur }}
-          eager={props.position < 2 ? "above-fold" : "below-fold"}
-          alt={props.story.headline || ""}
-        />
-      </figure>
-      ...
-      ...
-    </Link>
-  );
-}
-
-export function StoryGrid({ stories = [] }) {
-  ...
-  return (
-    <div className="story-grid">
-      {stories.map(story => (
-        <StoryGridStoryItem story={story} />
-      ))}
-    </div>
-  );
-}
-
-```
+   ```javascript
+   const [perfObj, setPerfObj] = useState({size: '5vw', blur: 10});
+
+   useEffect(() => {
+     setTimeout(() => {
```

Comment:
why do we have so many indentation changes ?

Reply:
I only arranged it properly, previously it was not indented properly